index3.js

```diff
@@ -0,0 +1,104 @@
+const readline = require("readline");
+const rl = readline.createInterface(process.stdin, process.stdout);
+
+function ask(questionText) {
+  return new Promise((resolve, reject) => {
+    rl.question(questionText, resolve);
+  });
+}
+
+function randomNum(min, max) {
+  return Math.floor(Math.random() * (max - min + 1) + min);
+}
+let min = 1;
+let max = 100;
+//gives you a random number in a range as solid integers (in this case 1-100)
+
+initialize();
+
+async function initialize() {
+  console.log("Which game would you like to play?");
+
+  let game = await ask(
+    "Would you like to pick or guess the number? \nType 'guess' or 'pick'. \n"
+  );
+  if (game === "guess") {
+    guess();
+  }
+  if (game === "pick") {
+    pick();
+  } 
+}
```

Comment:
This is a really clean and concise way to choose which way to play, and makes it easy to see what the steps are being taken in the code.  The one thing I will say, is there is no room for error when picking the game type.  If I make a spelling error, the game breaks.  There are more spots like this further down, so I won't mention it each time, but error handling is really important, so the game doesn't just break on user error.  There are a lot of ways to approach this, but one way would be to have a function that checks against a list of available inputs, and makes them ask again if they didn't input something the game expects.